### PR TITLE
testing/skopeo: new aport

### PR DESCRIPTION
**Pull request by errm**

### CODE REVIEW

File: testing/skopeo/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# Maintainer: Ed Robinson <ed@reevoo.com>
+pkgname=skopeo
+pkgver=0.1.23
+pkgrel=0
+pkgdesc="Work with remote images registries - retrieving information, images, signing content"
+url="https://github.com/projectatomic/skopeo"
+arch="x86_64"
+license="ASL 2.0"
+makedepends="go btrfs-progs-dev gnupg gpgme-dev ostree-dev lvm2-dev bash"
+options="!strip"
```
Comment (Shizmob) [inline testing/skopeo/APKBUILD]: Why is this needed? Can it be commented?

File: testing/skopeo/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Contributor: Ed Robinson <ed@reevoo.com>
+# Maintainer: Ed Robinson <ed@reevoo.com>
+pkgname=skopeo
+pkgver=0.1.23
+pkgrel=0
+pkgdesc="Work with remote images registries - retrieving information, images, signing content"
+url="https://github.com/projectatomic/skopeo"
+arch="x86_64"
```
Comment (Shizmob) [inline testing/skopeo/APKBUILD]: Does this work for other arches too?

File: testing/skopeo/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+build() {
+	cd ${_projectatomic}/${pkgname}
+	make binary-local
+}
+
+package() {
+	cd "$builddir"
+	install -Dm755 skopeo "$pkgdir"/usr/bin/skopeo
+	install -Dm644 LICENSE "$pkgdir"/usr/share/licenses/$pkgname/LICENSE
```
Comment (Shizmob) [inline testing/skopeo/APKBUILD]: We generally aim to avoid installing licenses if they are standard ones, afaik.

File: testing/skopeo/APKBUILD (hunk 1 above)
Comment (errm) [inline testing/skopeo/APKBUILD]: because this is go, you cannot strip the binary in the normal way...

Comment (errm) [inline testing/skopeo/APKBUILD]: IIRC this is just set without comment on every other go based package

File: testing/skopeo/APKBUILD (hunk 2 above)
Comment (errm) [inline testing/skopeo/APKBUILD]: It should do, but I have no way of testing, I only have an Intel machine to test on.

File: testing/skopeo/APKBUILD (hunk 3 above)
Comment (errm) [inline testing/skopeo/APKBUILD]: I am not a lawyer, but in my view not including the license in the package would break 4 (a) of ASL 2.0

We are installing the licence in some other packages IIRC

File: testing/skopeo/APKBUILD (hunk 1 above)
Comment (errm) [inline testing/skopeo/APKBUILD]: e.g. https://github.com/alpinelinux/aports/search?p=1&q=%21strip+go&type=&utf8=%E2%9C%93

Comment (jirutka) [inline testing/skopeo/APKBUILD]: We used to not strip go binaries because of some FUD, but then we realised that they can be normally striped as any other binary. However, it remained unchanged in existing abuilds. If you're not sure that it doesn't work stripped, then please remove `!strip`.

File: testing/skopeo/APKBUILD (hunk 3 above)
Comment (jirutka) [inline testing/skopeo/APKBUILD]: In any case we split licenses into a -doc subpackage (add `subpackages=$pkgname-doc`).

**Comment by errm** [comment]
@jirutka I have made some changes PTAL

**Comment by errm** [comment]
🤷‍♂️

**Comment by clandmeter** [comment]
Sorry but this seemed to have merged with new version of LXC.
